nixos/modules/services/backup/duplicati.nix

```diff
@@ -53,37 +53,81 @@ in
           Run as root with special care.
         '';
       };
+
+      parameters = lib.mkOption {
+        default = "";
+        type = lib.types.lines;
+        example = ''
+          --webservice-allowedhostnames=*
+        '';
+        description = ''
+          This option can be used to store some or all of the options given to the
+          commandline client.
+          Each line in this option should be of the format --option=value.
+          The options in this file take precedence over the options provided
+          through command line arguments.
+          <link xlink:href="https://duplicati.readthedocs.io/en/latest/06-advanced-options/#parameters-file">Duplicati docs: parameters-file</link>
+        '';
+      };
+
+      parametersFile = lib.mkOption {
+        default = null;
+        type = lib.types.nullOr lib.types.path;
+        description = ''
+          This file can be used to store some or all of the options given to the
+          commandline client.
+          Each line in the file option should be of the format --option=value.
+          The options in this file take precedence over the options provided
+          through command line arguments.
+          <link xlink:href="https://duplicati.readthedocs.io/en/latest/06-advanced-options/#parameters-file">Duplicati docs: parameters-file</link>
+        '';
+      };
     };
   };
 
-  config = lib.mkIf cfg.enable {
-    environment.systemPackages = [ cfg.package ];
+  config =
+    let
```

Comment:
Can you move these into the let block at the top of the module to reduce the module diff? Then this should be good to go, pinky promise